### ISSUE TITLE
Selected state colour fails accessibility check on some themes

### ISSUE THREAD

**Issue by timdavison** [misd-service-development/drupal-cambridge-theme#116]
![Screenshot 2020-07-31 at 14 12 52](https://user-images.githubusercontent.com/3321567/89038298-01cc0a80-d338-11ea-817e-8bafcb8878f5.png)

It needs a contrast of 4.5:1 with white text.

Theme 4: change selected state from #57831a to #578219 (so close!)
Theme 5: from #df671d to #bf5819
Theme 7: from #8c8989 to #797676
Theme 8: from 8f933b to #777b31
Theme 9: from #8c8989 to #797676
